modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -288,81 +429,156 @@ internal IntPtr GetPtr()
             return objectArray.GetPtr();
         }
 
+        /// <summary>
+        /// Converts this typed <see cref="Array{T}"/> to an untyped <see cref="Array"/>.
+        /// </summary>
+        /// <param name="from">The typed array to convert.</param>
         public static explicit operator Array(Array<T> from)
         {
             return from.objectArray;
         }
 
+        /// <summary>
+        /// Duplicates this <see cref="Array{T}"/>.
+        /// </summary>
+        /// <param name="deep">If true, performs a deep copy.</param>
+        /// <returns>A new Godot Array.</returns>
         public Array<T> Duplicate(bool deep = false)
         {
             return new Array<T>(objectArray.Duplicate(deep));
         }
 
+        /// <summary>
+        /// Resizes this <see cref="Array{T}"/> to the given size.
+        /// </summary>
+        /// <param name="newSize">The new size of the array.</param>
+        /// <returns><see cref="Error.Ok"/> if successful, or an error code.</returns>
         public Error Resize(int newSize)
         {
             return objectArray.Resize(newSize);
         }
 
+        /// <summary>
+        /// Shuffles the contents of this <see cref="Array{T}"/> into a random order.
+        /// </summary>
         public void Shuffle()
         {
             objectArray.Shuffle();
         }
 
+        /// <summary>
+        /// Concatenates these two <see cref="Array{T}"/>s.
+        /// </summary>
+        /// <param name="left">The first array.</param>
+        /// <param name="right">The second array.</param>
+        /// <returns>A new Godot Array with the contents of both arrays.</returns>
         public static Array<T> operator +(Array<T> left, Array<T> right)
         {
             return new Array<T>(left.objectArray + right.objectArray);
         }
 
         // IList<T>
 
+        /// <summary>
+        /// Returns the value at the given index.
+        /// </summary>
+        /// <value>The value at the given index.</value>
         public T this[int index]
         {
             get { return (T)Array.godot_icall_Array_At_Generic(GetPtr(), index, elemTypeEncoding, elemTypeClass); }
             set { objectArray[index] = value; }
         }
 
+        /// <summary>
+        /// Searches this <see cref="Array{T}"/> for an item
+        /// and returns its index or -1 if not found.
+        /// Optionally, the initial search index can be passed.
```

Comment:
This method does not take a `int from` parameter like the GDScript `find` method, so this description is incorrect.